instrumentation/spring/spring-webmvc-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springwebmvc/SpringWebMvcInstrumenters.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.springwebmvc;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import org.springframework.web.servlet.ModelAndView;
+
+public final class SpringWebMvcInstrumenters {
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.javaagent.spring-webmvc-3.1";
+
+  private static final Instrumenter<Object, Void> HANDLER_INSTRUMENTER;
+
+  private static final Instrumenter<ModelAndView, Void> MODEL_AND_VIEW_INSTRUMENTER;
+
+  static {
+    HANDLER_INSTRUMENTER =
+        Instrumenter.<Object, Void>newBuilder(
+                GlobalOpenTelemetry.get(), INSTRUMENTATION_NAME, new HandlerSpanNameExtractor())
+            .newInstrumenter(SpanKindExtractor.alwaysInternal());
+
+    MODEL_AND_VIEW_INSTRUMENTER =
+        Instrumenter.<ModelAndView, Void>newBuilder(
+                GlobalOpenTelemetry.get(),
+                INSTRUMENTATION_NAME,
+                new ModelAndViewSpanNameExtractor())
+            .addAttributesExtractor(new ModelAndViewAttributesExtractor())
+            .newInstrumenter(SpanKindExtractor.alwaysInternal());
```

Comment:
Nit: no need to specify INTERNAL, it's the default

```suggestion
            .newInstrumenter();

    MODEL_AND_VIEW_INSTRUMENTER =
        Instrumenter.<ModelAndView, Void>newBuilder(
                GlobalOpenTelemetry.get(),
                INSTRUMENTATION_NAME,
                new ModelAndViewSpanNameExtractor())
            .addAttributesExtractor(new ModelAndViewAttributesExtractor())
            .newInstrumenter();
```